Apromore-Extras/APMLogModule/src/main/java/org/apromore/apmlog/filter/typefilters/AttributeArcDurationFilter.java

```diff
@@ -74,7 +75,9 @@ private static boolean conformRule(ATrace trace, LogFilterRule logFilterRule) {
                                                                     double lowBoundVal, double upBoundVal) {
         List<Double> durList= new ArrayList<>();
 
-        List<AActivity> activityList = trace.getActivityList();
+        List<AActivity> activityList = trace.getActivityList().stream()
+                .filter(x -> trace.getValidEventIndexBitSet().get(x.getEventIndexes().get(0)))
```

Comment:
Similarly to the 7.19 version of this PR, is it possible for this code to fail if one of the trace activities returned by `trace.getActivityList()` has an empty list returned by `.getEventIndexes()`?

Reply:
If the trace has empty events/activities, it wouldn't be included in the log.